docs/src/user_docs/assembly/instruction_reference.md

```diff
@@ -307,10 +307,11 @@ High-level constructs for controlling the execution flow.
 
 Instructions for communicating with the host through events and tracing.
 
-| Instruction | Stack Input | Stack Output | Cycles | Notes                                                                                                                                    |
-| ----------- | ----------- | ------------ | ------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| `emit.<event_id>` | `[...]`     | `[...]`      | 5      | Emits an event with the specified `event_id` to the host. Does not change the state of the operand stack. The `event_id` can be any 32-bit value specified either directly or via a [named constant](./code_organization.md#constants). Events allow programs to communicate contextual information to the host for triggering appropriate actions. Example: `emit.123` or `emit.EVENT_ID_1` |
-| `trace.<trace_id>` | `[...]`     | `[...]`      | 0      | Emits a trace with the specified `trace_id` to the host. Does not change the state of the operand stack. The `trace_id` can be any 32-bit value specified either directly or via a [named constant](./code_organization.md#constants). Only active when programs are run with tracing flag (`-t` or `--trace`), otherwise ignored. Example: `trace.123` or `trace.TRACE_ID_1` |
+| Instruction        | Stack Input       | Stack Output | Cycles | Notes                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+|--------------------|-------------------|--------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `emit.<event_id>`  | `[...]`           | `[...]`      | 3      | Emits an event with the specified `event_id` to the host. The net effect on the operand stack is no change (internally expands to `push.<event_id> emit drop`). The `event_id` can be any 32-bit value specified either directly or via a [named constant](./code_organization.md#constants). Events allow programs to communicate contextual information to the host for triggering appropriate actions. Example: `emit.123` or `emit.EVENT_ID_1` |
```

Comment:
Similar to the other comment - I'd remove the "32-bit" part (I don't think we are enforcing this, are we?).